libs/core/allocator_support/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@
 # Distributed under the Boost Software License, Version 1.0. (See accompanying
 # file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
 
-cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.17 FATAL_ERROR)
```

Comment:
By the way, should we perhaps get rid of these?

Reply:
Yes I think it was needed when the modules were sub-`project()`s but looks like it's not the case anymore :+1:

Reply:
Didn't we plan to make modules available for external projects? I thought those would be needed for that.

Reply:
Yes, sometime in the very beginning of modularization. However, I'm still of the opinion that the modules are too fine-grained to expose directly to users (are we going to have a well-defined, public API for each module? versioning?). If that were to happen there would be a lot more work than just keeping the `cmake_minimum_required` calls in the modules, and adding those back when/if needed is not very difficult.